### ISSUE TITLE
Up/down icons next to down/up speeds are inaccessible to assistive technologies

### ISSUE THREAD

**Issue by Woebin** [Chocobozzz/PeerTube#7130]
### Describe the current behavior

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG22) [SC 1.1.1 (A) Non-text content](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content).

In the video player widget, there are up/down arrow icons along with numbers to indicate the current up/down transfer speeds, pictured below.
![Up/down arrows.](https://github.com/user-attachments/assets/92ad77aa-c753-43a9-8801-00f34f525f57)

These arrows are just coded as spans with classes, without any accessible names or other information available to assistive technologies.
![Up/down arrows and associated markup as shown in Chrome Devtools.](https://github.com/user-attachments/assets/1910c45d-365f-47f7-a44b-7f7607423abe)

The icons are completely hidden from assistive technologies, and the numbers/data units are presented without context. Screen reader users will just get the string "48 KB 0 B" in the instance pictured, and won't be made aware of what those numbers mean.

### Steps to reproduce

Inspect the icons in the browser's developer tools.

### Describe the expected behavior

The icons should be inside <img> tags with associated `alt` texts, e.g. `<img alt="Download speed:" src="...">`. Assistive technology users should be able to access the information these icons convey.

### Additional information

* PeerTube instance:
  * URL: https://peertube-eu.howlround.com/
  * Version: 7.2.1
  * NodeJS version: Unknown
  * Ffmpeg version: Unknown

* Browser name, version and platforms on which you could reproduce the bug: MacOS 15.5, Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).

**Comment by Chocobozzz** [comment]
Thanks! Should be fixed by https://github.com/Chocobozzz/PeerTube/commit/c958f6271f6c36dabcd2b7d5bc41c032483da781
